packages/app/src/systems/Chains/eth/containers/ManageEthAssetsDialog.tsx

```diff
@@ -0,0 +1,142 @@
+  }, [assets, newAssetAddress]);
+
+  const isValid = useMemo(() => {
+    return (
+      (newAssetAddress.length !== 0 &&
+        isAddress(newAssetAddress) &&
+        !doesAssetExist) ||
+      newAssetAddress.length === 0
+    );
+  }, [newAssetAddress, doesAssetExist]);
+
+  return (
+    <>
+      <Dialog.Heading>
+        <Box.Stack>
+          <Box.Flex justify="space-between" css={styles.dialogHeadingContainer}>
+            <Icon icon="ArrowLeft" onClick={store.openEthAssetsDialog} />
+            <Text color="intentsBase12" fontSize="sm">
+              Manage token list
+            </Text>
+            <Box>
+              <Dialog.Close />
+            </Box>
+          </Box.Flex>
+          <Form.Control isInvalid={!isValid}>
+            <Input size="md" css={styles.text}>
+              <Input.Field
+                placeholder="Paste custom address"
+                onChange={(e) => setNewAssetAddress(e.target.value)}
+                value={newAssetAddress}
+              />
+            </Input>
+            <Form.ErrorMessage>{`Please enter a valid address.${
+              doesAssetExist ? '  This address has already been added.' : ''
+            }`}</Form.ErrorMessage>
+          </Form.Control>
+          {!isError && !!data && isValid && (
+            <Button
+              size="sm"
+              onPress={() => {
+                addAsset({
+                  asset: {
+                    address: data?.address,
+                    decimals: data?.decimals,
+                    symbol: data?.symbol,
+                  },
+                });
+                store.openManageAssetsDialog();
+                setNewAssetAddress('');
+              }}
+            >
+              Add token
+            </Button>
+          )}
+          {isError && !!newAssetAddress.length && isValid && (
+            <Button
+              size="sm"
+              onPress={() =>
+                store.openAddAssetsDialog({
+                  assetAddress: newAssetAddress,
+                })
+              }
+            >
+              Add token
+            </Button>
+          )}
+        </Box.Stack>
+      </Dialog.Heading>
+      <Dialog.Description>
+        <CardList>
+          {assets.map((asset, i) => (
+            <CardList.Item
+              key={asset.address + asset.symbol + String(i)}
+              css={styles.cardListItem}
+            >
+              <Box.Flex justify="space-between" css={styles.actionButton}>
+                <Box.Flex gap="$2" align="center">
+                  <Image alt=" " src={asset.image} />
+                  {asset.symbol}
+                </Box.Flex>
+                {asset.address !== NativeAssetId && (
+                  <Icon
+                    icon="SquareRoundedX"
+                    onClick={() => removeAsset({ address: asset.address })}
+                    color="scalesRed10"
+                  />
+                )}
```

Comment:
if it's ETH asset..  `asset.address === NativeAssetId`, 

also show iconButton, but disabled and with a tooltip: 
"ETH is the native asset. It can't be removed."